docs/dev/concepts.rst

```diff
@@ -60,93 +42,74 @@ perform. Recipes contain three important pieces of information:
    the recipe's **arguments**.
 
-Recipes are stored in the Normandy database and the :ref:`runtime <runtime>`
-fetches them from the service during execution. Here's an example of a recipe in
-JSON format as returned by Normandy:
+Recipes are stored in the Normandy server's database and the Normandy client
+fetches them from the service during execution. Here's an example of a recipe
+in JSON format, as fetched by the Normandy client:
 
 .. code-block:: json
 
    {
       "id": 1,
       "name": "Console Log Test",
       "revision_id": 12,
-      "action": {
-         "name": "console-log",
-         "implementation_url": "https://normandy.cdn.mozilla.net/api/v1/action/console-log/implementation/8ee8e7621fc08574f854972ee77be2a5280fb546/",
-         "arguments_schema": {
-            "$schema": "http://json-schema.org/draft-04/schema#",
-            "description": "Log a message to the console",
-            "type": "object",
-            "properties": {
-               "message": {
-                  "default": "",
-                  "description": "Message to log to the console",
-                  "type": "string",
-               }
-            },
-            "required": [
-               "message"
-            ],
-         }
-      },
+      "action": "console-log",
       "arguments": {
          "message": "It works!"
-      }
+      },
+      "filter_expression": "normandy.version >= '65.0' && stableSample(0.5)"
    }
 
-Note that the recipe as returned by the API also contains data about the action,
-including a schema describing the arguments field.
+.. info::
+
+   The data retrieved by the Normandy client is the minimum subset of data
+   that is needed execute the action. Other API end points return more
+   information about the recipe, including it's history and approval status.
 
 .. _JEXL: https://github.com/TechnologyAdvice/Jexl
 .. _Telemetry: https://wiki.mozilla.org/Telemetry
 
 .. _runtime:
 
-Runtime
--------
-The runtime is an execution environment that can affect a running instance of
-Firefox. Upon activation (typically a few moments after Firefox launches), the
-runtime:
+Normandy Client
+---------------
+The Normandy client is `a Firefox component`_ that reads and executes
+instructions from the server. Upon activation (typically every few hours),
+the runtime:
 
 1. Downloads :ref:`recipes <recipes>` from the Normandy service.
 2. Verifies the signature of the recipes.
 3. Evaluates the recipe filters and filters out recipes that do not match the
    client the runtime is installed within.
-4. Downloads the :ref:`actions <actions>` for the remaining recipes.
-5. Executes the action code for each recipe in a sandbox, passing in the
-   arguments from the recipe, and a :doc:`driver object </dev/driver>` containing
-   methods that can perform privileged actions.
-
-The runtime is implemented as `a component of Firefox in mozilla-central`_.
+4. Passes the recipes to the corresponding action code for execution.
```

Comment:
This sentence is strongly situated within the point of view of the code itself. How about `Executes the corresponding action code using the recipe's arguments`? Details can be found in mozilla-central anyhow..